packages/frontend-api/src/Model/Token/JwtConfigurationFactory.php

```diff
@@ -14,6 +14,8 @@ class JwtConfigurationFactory
 {
     protected const FRONTEND_API_KEYS_FILEPATH_PARAMETER = 'shopsys.frontend_api.keys_filepath';
 
+    protected Configuration $configuration;
```

Comment:
the class should not be defined as service anymore, right? So the following lines should be removed from the `services.yaml`:

```
    Lcobucci\JWT\Configuration:
        factory: [ '@Shopsys\FrontendApiBundle\Model\Token\JwtConfigurationFactory', create ]
```

Reply:
btw, the change deserves upgrade notes IMHO (something like "Lcobucci\JWT\Configuration is not a service anymore, use Shopsys\FrontendApiBundle\Model\Token\JwtConfigurationFactory::create() to get the configuration")